tslib/src/functions/check-notifier/check-notifier.data.service.ts

```diff
@@ -21,14 +36,11 @@ export class CheckNotifierDataService implements ICheckNotifierDataService {
     }
     const checkRequest: ITransactionRequest = {
       sql: `UPDATE [mtc_admin].[check]
-      SET checkStatus_id=@checkStatusId
+      SET checkStatus_id=(SELECT cs.id FROM
+        [mtc_admin].[checkStatus] cs
+        WHERE cs.code='CMP')
```

Comment:
If we are setting the status to complete, should we also set `pupil.checkComplete`, `check.complete`, `check.completedAt` ?